src/alda/repl/commands/ReplInfo.java

```diff
@@ -0,0 +1,103 @@
+package alda.repl.commands;
+
+import java.util.List;
+import java.util.function.Consumer;
+import java.util.stream.Collectors;
+
+import com.google.gson.Gson;
+import com.google.gson.JsonArray;
+import com.google.gson.JsonObject;
+import com.google.gson.reflect.TypeToken;
+import com.google.gson.JsonParseException;
+
+import alda.AldaResponse.AldaScore;
+import alda.AldaServer;
+import alda.error.NoResponseException;
+import alda.error.ParseError;
+import jline.console.ConsoleReader;
+
+public class ReplInfo implements ReplCommand {
+  
+  private Gson gson = new Gson();
+	
+  @Override
+  public void act(String args, StringBuffer history, AldaServer server,
+                  ConsoleReader reader, Consumer<AldaScore> newInstrument)
+                		  throws NoResponseException {
+	try {
+	  String res = server.parseRaw(history.toString(), "data");
+	  if(res == null) {
+	    System.err.println("An internal error occurred when reading the score.");
```

Comment:
I think the indentation in this file is a bit wonky, this line has tabs and spaces.

Reply:
You're right - my laziness :-P

Reply:
Fixed.